zcash_extras/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "zcash_extras"
+version = "0.1.0"
+edition = "2018"
+
```

Comment:
Please add
```
[lib]
doctest = false
```
as there is failing doc test for `create_spend_to_address`.

Reply:
thanks done